courses/free-courses-en.md

```diff
@@ -223,6 +223,7 @@
 
 * [Bash Basics for Cloud Computing](https://www.udemy.com/course/bash-basics-for-cloud-computing/) - Kumulus Technologies (Udemy)
 * [Bash Scripting Full Course 3 Hours](https://www.youtube.com/watch?v=e7BufAVwDiM) - Linuxhint
+* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag&t=569s) - Joe Collins
```

Comment:
Please remove the timestamp from the link

Reply:
removed the time stamps. Thanks for reviewing !